rotkehlchen/api/rest.py

```diff
@@ -3196,3 +3196,61 @@ def purge_gitcoin_grant_data(self, grant_id: Optional[int]) -> Response:
             self.rotkehlchen.data.db, self.rotkehlchen.msg_aggregator,
         ).delete_gitcoin_ledger_actions(grant_id)
         return api_response(OK_RESULT, status_code=HTTPStatus.OK)
+
+    def add_manual_price(  # pylint: disable=no-self-use
+        self,
+        from_asset: Asset,
+        to_asset: Asset,
+        price: Price,
+        timestamp: Timestamp,
+    ) -> Response:
+        historical_price = HistoricalPrice(
+            from_asset=from_asset,
+            to_asset=to_asset,
+            source=HistoricalPriceOracle.MANUAL,
+            timestamp=timestamp,
+            price=price,
+        )
+        GlobalDBHandler().add_single_historical_price(historical_price)
+        return api_response(OK_RESULT, status_code=HTTPStatus.OK)
+
+    def edit_manual_price(  # pylint: disable=no-self-use
+        self,
+        from_asset: Asset,
+        to_asset: Asset,
+        price: Price,
+        timestamp: Timestamp,
+    ) -> Response:
+        historical_price = HistoricalPrice(
+            from_asset=from_asset,
+            to_asset=to_asset,
+            source=HistoricalPriceOracle.MANUAL,
+            timestamp=timestamp,
+            price=price,
+        )
+        edited = GlobalDBHandler().edit_manual_price(historical_price)
+        if edited:
+            return api_response(OK_RESULT, status_code=HTTPStatus.OK)
+        return api_response(
+            result={'result': False, 'message': ''},
+            status_code=HTTPStatus.OK,
```

Comment:
Ehm ... no? The status should not be okay. It should be `HTTPStatus.CONFLICT` since there was a conflict at addition.